packages/coding-standards/.phpcs.xml.dist-sample

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<ruleset name="Jetpack-Plugin">
+	<config name="minimum_supported_wp_version" value="5.3" />
+	<config name="testVersion" value="5.6-"/>
+
+	<rule ref="Jetpack"/>
+
+	<rule ref="VariableAnalysis.CodeAnalysis.VariableAnalysis">
+		<!-- Template files use extracted variables. -->
+		<exclude-pattern>/extensions/blocks/podcast-player/templates/*</exclude-pattern>
+	</rule>
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<property name="text_domain" type="array" value="jetpack" />
```

Comment:
This is the old style of property declaration. If you support PHPCS 3.3.0 and above, then use the new style.

```xml
<rule ref="WordPress.WP.I18n">
	<properties>
		<property name="text_domain" type="array">
			<element value="jetpack"/>
		</property>
	</properties>
</rule>
```